src/components/ComputeModal.js

```diff
@@ -305,6 +304,8 @@ export const ComputeModalBase = ({ onDismiss, onSuccess, runtimes, persistentDis
 
   const canUpdateNumberOfWorkers = () => !currentRuntimeDetails || isRuntimeRunning()
 
+  const requiresStandard = () => getToolForImage(_.find({ image: selectedLeoImage }, leoImages)?.id) === tools.RStudio.label
```

Comment:
nit: maybe rename this to `requiresGCE`. People may have different perspectives on what's standard, especially if they're coming into this code without too much background knowledge of google compute

Reply:
I agree that `requiresStandard` can be ambiguous.

One downside of `requiresGCE` is that it is Google Cloud Platform (GCP)-specific, which would renaming when we introduce Azure but I guess at that point, that'd be the least of renamings we'd have to do :)

Nit: I like `requiresGce` better for consistency with how we've used `gce` in variable/method names.